server/images.go

```diff
@@ -815,14 +821,14 @@ func formatParams(params map[string][]string) (map[string]interface{}, error) {
 						return nil, fmt.Errorf("invalid float value %s", vals)
 					}
 
-					out[key] = floatVal
+					out[key] = float32(floatVal)
```

Comment:
`ParseFloat` returns a float64. Since this is explicitly handling float32s, type convert it